dali/ft/daftformat.cpp

```diff
@@ -1339,81 +1327,67 @@ void CUtfQuickPartitioner::findSplitPoint(offset_t splitOffset, PartitionCursor
 {
     const byte *buffer = bufferBase();
     numInBuffer = bufferOffset = 0;
-    if (splitOffset != 0)
```

Comment:
git not making it easy to see the change, but I see it's just removal of this if and the else block.
@ghalliday - would you look at this change too please?